index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Abel portfolio</title>
+  </head>
+  <body>
+    <!-- Navbar -->
+    <nav class="nav">
+      <logo>
+        <p class="logo">Abel Tsegaye</p>
+      </logo>
```

Comment:
- [ ] Kindly make sure to wrap the logo into the Anchot `a` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.